src/Datadog.Sketches/Stores/DenseStore.cs

```diff
@@ -0,0 +1,351 @@
+// <copyright file="DenseStore.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+// </copyright>
+
+#pragma warning disable SA1600 // Elements should be documented
```

Comment:
You even _did_ document everything here, you've become drunk with power 😂

Reply:
No there are a few protected methods/constructors that are not documented 😛

Reply:
Oh, you're requiring documenting _protected_ members too? That's hardcore